docs/blog/newsletter-february-2020.rst

```diff
@@ -0,0 +1,96 @@
+.. post:: February 03, 2020
+   :tags: newsletter
+
+#########################################
+Write the Docs Newsletter – February 2020
+#########################################
+
+Happy 2020, documentarians! It's Beth and the newsletter crew here, with the latest and greatest from the Write the Docs community.
+
+Two bits and pieces for you on the news front today. In December we annouced dates for the Prague conference, which will be 13-15 September - and in a brand shiny new venue, the Hotel Olšanka! `Read all about it here </conf/prague/2020/news/init/>`__. As for across the pond, the speakers for Portland speakers will be announced soon, so keep your eyes peeled for those. The conference is 3-5 May, and `tickets are on sale right now </conf/portland/2020/tickets/>`__.
+
+So with that, let's see what's been cooking on Slack recently!
+
+-----------------------------
+#learn-tech-writing book club
+-----------------------------
+
+There's been a buzz of activity all through January in the `#learn-tech-writing <https://app.slack.com/client/T0299N2DL/C7YJR1N02>`__ channel, where folks have been discussing *The Product is Docs* chapter-by-chapter. As those involved have kindly started an effort to write up all the fantastic conversations on the topic, we're not going to write it up in the newsletter, but we wanted to point you towards what's been written up so far:
+
+- `Agile and technical writing <https://github.com/writethedocs/www/blob/bookClubWriteUps/docs/book-club/splunk-product-docs/2-agile.rst>`__
+- `Documenting third-party products <https://github.com/writethedocs/www/commit/89ead30998101c1ce1038203c74e53aa50719a29>`__
+- `Technical editing <https://github.com/writethedocs/www/blob/bookClubWriteUps/docs/book-club/splunk-product-docs/13-technical-editing.rst>`__
+
+If you'd like to help out with write-ups, drop into the channel!  As a bonus, they're looking into what to discuss next, so if you have a book you think would be great for group discussions, go ahead and make the suggestion. 
+
+---------------------------------------------
+Supporting documentation, documenting support
+---------------------------------------------
+
+A recent question about how to balance docs work and support content produced some interesting responses. The original post assumed two distinct teams, one for documentation and one for content produced by a support team, with each team producing distinct content. This is evidently the situation in many companies. For example, one poster described a workflow where answers to frequently-asked support questions are added to the documentation, but if a potential docs page seems particularly complicated, or addresses an edge case, the writer asks support to document it. This story describes distinct teams -- Docs and Support -- and distinct types of content.
```

Comment:
```suggestion
A recent question about how to balance docs work and support content produced some enthusiastic discussion. The original post assumed two separate teams, one for documentation and one for content produced by a support team, with each team producing their own content. This is evidently the situation in many companies. For example, one poster described a workflow where answers to frequently-asked support questions are added to the documentation, but if a potential docs page seems particularly complicated, or addresses an edge case, the writer asks support to document it. This story describes distinct teams -- Docs and Support -- and distinct types of content.
```

Reply:
Something like this? Edit as needed. Also 

> The original post assumed two separate teams, one for documentation and one for content produced by a support team

reads super odd to me. One for docs and one for support.

Reply:
eep, yes. fixed